test/install_script/nvm_install_with_aliased_dot

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
why does this one need to change?

Reply:
It's because of the `shopt` bash builtin being used. Here's the result of running that test suite without this:
```
  ✗ nvm_install_with_aliased_dot
setup:1: command not found: shopt
nvm
cleanup:3: command not found: shopt
```